### PR TITLE
I added a new controller and a new view where you can see a name in t…

### PR DESCRIPTION
**Pull request by pablohincapie**
## Summary
Greeting on the browser

## Trello Card
https://trello.com/c/HwvfQJw0/17-crear-un-controller-y-una-vista-de-bienvenida

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
-*.class
+HELP.md
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**/build/
+!**/src/test/**/build/
 
-# Log file
-*.log
```
Comment (AleFalcon) [inline .gitignore]: I thnik that the logs should not upload

File: .gitignore [hunk 2]
```diff
@@ -1,23 +1,38 @@
-# Compiled class file
```
Comment (Gianni-Nicolas) [inline .gitignore:1]: Add them to git ignore

File: .gitignore [hunk 3]
```diff
@@ -1,23 +1,38 @@
...
-*.class
```
Comment (Gianni-Nicolas) [inline .gitignore:2]: Add them to git ignore

File: .gitignore [hunk 4]
```diff
@@ -1,23 +1,38 @@
...
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+bin/
+!**/src/main/**/bin/
+!**/src/test/**/bin/
 
-# BlueJ files
```
Comment (Gianni-Nicolas) [inline .gitignore]: Add them to git ignore

File: .gitignore [hunk 5]
```diff
@@ -1,23 +1,38 @@
...
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+out/
+!**/src/main/**/out/
+!**/src/test/**/out/
 
-# Mobile Tools for Java (J2ME)
```
Comment (Gianni-Nicolas) [inline .gitignore]: Add them to git ignore

File: .gitignore [hunk 6]
```diff
@@ -1,23 +1,38 @@
...
-.mtj.tmp/
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
 
-# Package Files #
```
Comment (Gianni-Nicolas) [inline .gitignore]: Add them to git ignore

File: .gitignore [hunk 7]
```diff
@@ -1,23 +1,38 @@
...
-*.jar
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
```
Comment (Gianni-Nicolas) [inline .gitignore:22]: Add them to git ignore

**Review by Gianni-Nicolas — DISMISSED** [review]
Great job Pablo, I just left you some comments related to git ignore. All extensions that are marked as removed should be included in the .gitignore file. Sure they were erased when we resolved the conflicts 😄

File: .gitignore (hunk 1 above)
Comment (pablohincapie) [reply .gitignore]: I added this line on gitignore

File: .gitignore (hunk 2 above)
Comment (pablohincapie) [reply .gitignore:1]: I added this line on gitignore

File: .gitignore (hunk 4 above)
Comment (pablohincapie) [reply .gitignore]: I added this line on gitignore

File: .gitignore (hunk 5 above)
Comment (pablohincapie) [reply .gitignore]: I added this line on gitignore

File: .gitignore (hunk 6 above)
Comment (pablohincapie) [reply .gitignore]: I added this line on gitignore

File: .gitignore (hunk 7 above)
Comment (pablohincapie) [reply .gitignore:22]: I added this line on gitignore

**Review by AleFalcon — APPROVED** [review]

**Review by Gianni-Nicolas — APPROVED** [review]

**Pull request merged**